google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TableInfo.java

```diff
@@ -399,6 +414,14 @@ public BigInteger getNumRows() {
     return labels.userMap();
   }
 
+  /**
+   * Returns true if a partition filter (that can be used for partition elimination) is
```

Comment:
@pmakani - hey i can't seem to get this right on GH... could you please see to getting the comment here right to be "is required for..."